variables.tf

```
@@ -115,3 +80,21 @@ variable "transition_to_ia" {
   description = "Indicates how long it takes to transition files to the IA storage class. Valid values: AFTER_7_DAYS, AFTER_14_DAYS, AFTER_30_DAYS, AFTER_60_DAYS and AFTER_90_DAYS"
   default     = ""
 }
+
+variable "transition_to_primary_storage_class" {
+  type        = string
+  description = "Describes the policy used to transition a file from infequent access storage to primary storage. Valid values: AFTER_1_ACCESS."
+  default     = ""
```

Make `transition_to_primary_storage_class`  a list of string and do the same for `transition_to_ia` above.
```suggestion
  type        = list(string)
  description = "Describes the policy used to transition a file from infequent access storage to primary storage. Valid values: AFTER_1_ACCESS."
  default     = []
  validation {
    condition = (
      length(var.transition_to_primary_storage_class) == 1 ? var.transition_to_primary_storage_class[0] == "AFTER_1_ACCESS" : length(var.transition_to_primary_storage_class) == 0
      )
    error_message = "Var `transition_to_primary_storage_class` must either be empty list or `[\"AFTER_1_ACCESS\"]`"
    }
```